action.yml

```diff
@@ -20,13 +20,28 @@ outputs:
 runs:
   using: 'composite'
   steps:
-    - uses: thebrowsercompany/check-python-exists@7a9d28cb86e2aae6d6ef75af09b4bb14fcd03ff1
+    - uses: thebrowsercompany/check-python-exists@enable-input-version
```

Comment:
Self-blocking: needs https://github.com/thebrowsercompany/check-python-exists/pull/6 to land at which point we can point to a specific commit on `main`.